webhook/stats_reporter.go

```diff
@@ -101,9 +101,9 @@ func (r *reporter) ReportRequest(req *admissionv1beta1.AdmissionRequest, resp *a
 		return err
 	}
 
-	metrics.Record(ctx, requestCountM.M(1))
-	// Convert time.Duration in nanoseconds to milliseconds
-	metrics.Record(ctx, responseTimeInMsecM.M(float64(d/time.Millisecond)))
+	metrics.RecordBatch(ctx, requestCountM.M(1),
+		// Convert time.Duration in nanoseconds to milliseconds
+		responseTimeInMsecM.M(float64(d/time.Millisecond)))
```

Comment:
It's strange that we're using a "float" here but our units are 1/1000 of a "second".

Probably not a fix for this PR, but it makes me slightly sad.

Reply:
yeah. changed to d.Milliseconds() anyway.